README.md

```diff
@@ -45,24 +45,26 @@ The main UI has some known limitations that are documented in code and in some p
 To build for .NET Framework 4.8, .NET 6.0, or .NET 7.0 (all Windows only), ensure that the .NET 7.0 SDK (or later) is installed and included in your PATH. Then, run the following commands from command prompt, Powershell, or Terminal:
 
 ```
-dotnet build MPF\MPF.csproj --framework [net48|net6.0-windows|net7.0-windows] --runtime [win-x64]
+dotnet build MPF\MPF.csproj --framework [net48|net6.0-windows|net7.0-windows] --runtime win-x64 --self-contained
```

Comment:
`--self-contained` isn't strictly necessary for building. It's a convenience for distribution.

Reply:
Eh, leaving it as-is is fine then.